docs/components/radio/index.html

```diff
@@ -69,6 +69,7 @@ <h2 id="basic-radio">Basic Radio</h2>
             :checked="isChecked"
             :disabled="isDisabled"
             :required="isRequired"
+            style="display: none;"
```

Comment:
Is this something that is needed?

Reply:
likely not... we shouldn't have any inline `style` attributes

Reply:
![Screenshot from 2019-10-28 15-56-36](https://user-images.githubusercontent.com/17586567/67675384-00cf0c80-f9a5-11e9-8d5d-4df36812c51b.png)
-------------------------------------------------

If i don't add display:none , we can see two radio checks as shown in attachment.

Reply:
If there are two radios showing, then there's a change that's happened elsewhere. Adding a `style="display: none;" in the HTML is an anti-pattern that's treating a symptom, rather than correcting the actual problem.

Reply:
Sure enough, the helper that visually hides the native radio was not applied. https://github.com/HelixDesignSystem/helix-ui/pull/625#discussion_r339645088

Reply:
Changes are done ... please review